app/models/concerns/findings/sort_columns.rb

```diff
@@ -119,16 +119,16 @@ def readings_desc_options
         reading_user = "COUNT(#{Reading.table_name}.user_id)"
         finding_user = "COUNT(#{FindingAnswer.table_name}.user_id)"
 
-        order_by_readings = "CASE \n"
-        order_by_readings << "WHEN (#{reading_user} < #{finding_user}) then (#{finding_user} - #{reading_user}) \n"
-        order_by_readings << "ELSE 0 \n"
-        order_by_readings << 'END DESC'
-        order_by_readings << ", #{quoted_table_name}.id DESC"
+        order_by_readings = "readings_count DESC, #{quoted_table_name}.id DESC"
```

Comment:
Lo mismo, pondría algo así:

```suggestion
        order_by_readings = "readings_count DESC, #{quoted_table_name}.#{qcn 'id'} DESC"
```